helpers/match_manipulator.py

```diff
@@ -25,13 +26,29 @@ def postUpdateHook(cls, matches):
         Send push notifications to subscribed users
         Only if the match is part of an active event
         '''
+        unplayed_matches = []
         for match in matches:
-            if match.event.get().now:
+            event = match.event.get()
+            if event.now and match.has_been_played:
                 logging.info("Sending push notifications for "+match.key_name)
                 try:
                     NotificationHelper.send_match_score_update(match)
                 except Exception, exception:
                     logging.error("Error sending match updates: "+str(exception))
+                    logging.error(traceback.format_exc())
+            elif not match.has_been_played and not event in unplayed_matches:
+                unplayed_matches.append(event)
+
+        
+        '''
+        If we have an unplayed match, send out a schedule update notification
+        '''
+        for event in unplayed_matches:
```

Comment:
`unplayed_matches` is an odd variable name. `unplayed_match_events` makes more sense. There's probably a better name for it, but I can't think of one at the moment.